packages/ui/resolution/ResolutionDecorator.js

```diff
@@ -76,6 +99,8 @@ const ResolutionDecorator = hoc(defaultConfig, (config, Wrapped) => {
 
 		constructor (props) {
 			super(props);
+			riConfig.fontSizeHandling = config.fontSizeHandling;
```

Comment:
Why don't you pass these configs to `init` function as parameters instead of setting this here?